### ISSUE TITLE
push "route-metric 10" invalid

### ISSUE THREAD

**Issue by huangshaokun** [OpenVPN/openvpn#574]
openvpn Connect can push route metric successfully, but openvpn gui does not push route metric successfully.

**Comment by schwabe** [comment]
Please include more details.

**Comment by huangshaokun** [comment]
openvpn Connect log 
![openvpn](https://github.com/OpenVPN/openvpn/assets/31309235/f2d36cc4-cf17-49fc-8fb0-c69a045a71fd)


openvpn gui log
![0CCAA6~1](https://github.com/OpenVPN/openvpn/assets/31309235/992c29e0-7234-4c46-a3a7-23aa9e555a2c)

**Comment by schwabe** [comment]
Sure. But what is your actual problem? That is just a diffrence in logs between two completely different implementations. Is there an actual problem you are trying to solve or are you just pointing out a difference in implementation/logging?

**Comment by huangshaokun** [comment]
Of course, I'm asking an actual question, not the log, just because I found this log when checking the problem.
Metric does not take effect, causing domain name resolution to still go to the original network (meaning that the dns set by openvpn does not take effect)

This is my configuration in openvpn.conf
push "dhcp-option DNS 192.168.5.1"
push "route-metric 10"

When using openvpn Connect as the client, the domain name resolution request is 192.168.5.1
But when I use openvpn gui as the client, the domain name resolution is not 192.168.5.1, but the dns server configured in the local network.

During the inspection, it was found that the metric did not take effect when using openvpn gui.

When using openvpn gui as the client, I manually changed the metric to 10 on win10 and it became normal.





During the test, I used nslookup to check which server the domain name resolution requested, and route print to check the metric of the route.
